Makefile

```diff
@@ -12,7 +12,7 @@ build:
 	cargo build
 
 test:
-	cargo run --bin cairo-test -- --starknet --path $(SOURCE_FOLDER)
+	cargo run --bin cairo-test -- --starknet $(SOURCE_FOLDER)
```

Comment:
is this a `1.0.0-alpha.7` change?

Reply:
Post alpha 7: https://github.com/starkware-libs/cairo/commit/3f4da6c0c9aa1672da4be73f10b6e386480c0455